### DISCUSSION TITLE
bg image forces line break on header/footer

### DISCUSSION THREAD

**Discussion by jkabisch** [marp-team/marp#424] [Q&A]
First of all: I just started moving to marp from latex beamer and already like it a lot! Excellent work!!!

I have the following problem:

When rendering the following markdown code using marp file.md

`
----
header: '1. Cells and components 2. Unicellular vs Multicellular 3. Archae Eubacteria'
---

## A cell is a cell is a cell
 (lat. *cella, cellular*: basement, small room)
 - all living organisms are comprised of cells
![width:600px bg right drop-shadow](https://upload.wikimedia.org/wikipedia/commons/thumb/1/11/Animal_Cell.svg/1024px-Animal_Cell.svg.png) 

`

the header and footer only have the space available the background image is not placed. A line break occurs within the header/footer. 

The intention is to use the bg feature to get column like behavior, so I can elaborate on images shown.

Would it be possible to not have this line break?

**Comment by yhatt** [comment]
Could you share the Markdown code as the code block, to make it easier for Marp community helping you? Wrap the Markdown code with triple-backticks.

````markdown
```markdown
# Put your Markdown here
```
````

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks
